tests/ut/ir/transforms/test_memory_reuse.py

```
@@ -1304,22 +1297,16 @@ def main(
-                    next_1: pl.Tile[[64, 64], pl.FP32, pl.MemRef(mem_vec_6, 0, 16384), pl.Mem.Vec] = (
+                    next_1: pl.Tile[[64, 64], pl.FP32, pl.MemRef(mem_vec_3, 0, 16384), pl.Mem.Vec] = (
                         pl.tile.add(extra_1, acc_1)
                     )
-                    next_0_mv: pl.Tile[[64, 64], pl.FP32, pl.MemRef(mem_vec_2, 0, 16384), pl.Mem.Vec] = (
-                        pl.tile.move(next_0, target_memory=pl.Mem.Vec)
-                    )
-                    next_1_mv: pl.Tile[[64, 64], pl.FP32, pl.MemRef(mem_vec_3, 0, 16384), pl.Mem.Vec] = (
-                        pl.tile.move(next_1, target_memory=pl.Mem.Vec)
-                    )
-                    out_0, out_1 = pl.yield_(next_0_mv, next_1_mv)
+                    out_0, out_1 = pl.yield_(next_0, next_1)
```

Ruff (0.15.10)

[warning] Unpacked variable `out_1` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)